doc/tier2.md

```diff
@@ -144,6 +144,49 @@ spec:
       credentials: ecs-credentials
 ```
 
+#### Update ECS Credentials
+
+There might be an operational need to update ECS credentials on a regular basis, where the following steps could be taken:
+
+1. In case Pravega Cluster has not yet been deployed, replace "secretKey" and/or "identity" with new values inside the Pravega Yaml (or its equivalent)
+    ```
+    ...
+    spec:
+      tier2:
+        ecs:
+          https://10.243.86.64:9021?namespace=namespace%26identity=oldUser%26secretKey=oldPassword
+          bucket: "shared"
+          prefix: "example"
+    ```
+2. If Pravega Cluster has already been deployed, then follow these steps:
+
+    1. Modify Segmentstore configmap, find "EXTENDEDS3_CONFIGURI", and then replace "secretKey" and/or "identity" with new values
+        ```
+        $ kubectl edit configmap pravega-pravega-segmentstore
+        ```
+
+        ```
+        ...
+        EXTENDEDS3_BUCKET: shared
+        EXTENDEDS3_CONFIGURI: https://10.243.86.64:9021?namespace=namespace%26identity=oldUser%26secretKey=oldPassword
+        EXTENDEDS3_PREFIX: example
+        ...
+        ```
+    2. Delete all (running) Segmentstore pod(s) in one of the two approaches below:
+        ```
+        $ kubectl delete po -l component=pravega-segmentstore
+        ```
+        ```
+        $ kubectl delete po pravega-pravega-segmentstore-1
+        $ kubectl delete po pravega-pravega-segmentstore-2
+        $ kubectl delete po pravega-pravega-segmentstore-3
+        ...
+        ```
+        As StatefulSet, new Segementstore pods will be automatically created with the new ECS credentials,
+        since the default upgrade strategy of Segmentstore is `OnDelete` instead of `RollingUpdate`.
+
+        In case ECS gives "grace period" when both old and new credentials are accepted, Pravega service is technically uninterrupted during the above process.
```

Comment:
> In case ECS gives....
This doesn't sound that ECS supports grace period. Make it affirmative statement since ECS supports grace period

Reply:
Fixed.